src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -286,6 +286,36 @@ public static string GetAzureWebsiteUniqueSlotName(this IEnvironment environment
             return name?.ToLowerInvariant();
         }
 
+        /// <summary>
+        /// Gets the Antares version.
+        /// </summary>
+        public static string GetAntaresVersion(this IEnvironment environment)
+        {
+            if (environment.IsLinuxAzureManagedHosting())
+            {
+                return environment.GetEnvironmentVariableOrDefault(AntaresPlatformVersionLinux, string.Empty);
+            }
+            else
+            {
+                return environment.GetEnvironmentVariableOrDefault(AntaresPlatformVersionWindows, string.Empty);
+            }
+        }
+
+        /// <summary>
+        /// Gets the Instance id.
+        /// </summary>
+        public static string GetInstanceId(this IEnvironment environment)
```

Comment:
For these 3 new properties, can you add some tests to EnvironmentExtensionTests to verify the platform specific logic? See examples in that test suite. For each property, we can have a Theory test that verifies platform/result combinations

Reply:
Added tests.